package.json

```diff
@@ -8,9 +8,10 @@
     "packages/*"
   ],
   "scripts": {
-    "postinstall": "lerna run clean && lerna run build",
+    "postinstall": "yarn clean && lerna run build",
+    "clean": "rm -rf ./packages/*/lib ./packages/*/*.tsbuildinfo",
```

Comment:
should we use rimraf instead everywhere? It would be nice to support developing lodestar on windows?

Reply:
Can we tackle that in another PR? I'm sure there are more places with unix syntax